regen/embed.pl

```diff
@@ -304,7 +304,7 @@ sub hide {
     my ($from, $to, $indent) = @_;
     $indent = '' unless defined $indent;
     my $t = int(length("$indent$from") / 8);
-    "#${indent}define $from" . "\t" x ($t < 3 ? 3 - $t : 1) . "$to\n";
+    "#${indent}define $from" . "    " x ($t < 3 ? 3 - $t : 1) . "$to\n";
```

Comment:
This and the two similar cases below might want to be something closer to (untested):
```
  my $t = length("$indent$from");
  "#${indent}define $from" . " " x ($t < $my_indent ? $my_indent - $t : 1) . "$to\n";
```